docs/mp/restclient/09_rest-client.adoc

```diff
@@ -113,8 +209,12 @@ interface GreetRestClient {
 
 }
 ----
-Then create runnable method the same way as it is described in
-`Creating new client - Interface`,but with baseUri `http://localhost:8080/greet`
-and the above interface.
+Then create a runnable method as described in `Creating new client - Interface`,but with baseUri `http://localhost:8080/greet` and the above interface.
```

Comment:
The `Creating new client - Interface` should be a link if it refers to an actual document. If it does not, then the paragraph does not make sense